s4e-web/src/app/views/map-view/view-manager/overlay-list-modal/overlay-list-modal.component.html

```diff
@@ -81,7 +81,7 @@
                   <ng-container *ngIf="hasErrors('url')" >
                     <span
                       class="error"
-                      *ngFor="let error of newOverlayForm.controls.url.errors | keyvalue"
+                      *ngFor="let error of filterWmsUrlErrors(newOverlayForm.controls.url.errors) | keyvalue"
```

Comment:
`filterWmsUrlErrors` should be either a pipe or a stream which is `async` ed.
The methrod here will be called on every update cycle which is an antipattern